test/assertions_test.lua

```diff
@@ -1,7 +1,7 @@
 local t = require('luatest')
 local g = t.group()
 
-local helper = require('test.helper')
+local helper = require('test.helpers.helper')
```

Comment:
Nit: helpers.helper is a bit repetitive. Especially because it is the only helper :)